ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -93,6 +100,14 @@ const LoadWorkspaceDataForm = React.memo(({ namespace, name }) =>
       size="small"
       fields={FORM_FIELDS}
     />
+    <p>
+      Need help? please submit &nbsp;
+      <a href="https://github.com/broadinstitute/seqr/issues">GitHub Issues</a>
```

Comment:
We want to direct them to the page for submitting a new bug report, not the page for the list of issues: `https://github.com/broadinstitute/seqr/issues/new?labels=bug&template=bug_report.md`